### ISSUE TITLE
android: "Disable animations" isn't working for animated emoji

### ISSUE THREAD

**Issue by chrisbobbe** [zulip/zulip-flutter#1925]
A user in a closed community on Zulip Cloud reports that the Android app doesn't respect the system setting to disable animations. (We're supposed to be showing still versions of animated emojis when that setting is on.) (Link for those with access: [#zulip > Animated Reactji @ 💬](https://recurse.zulipchat.com/#narrow/channel/19176-zulip/topic/Animated.20Reactji/near/543588969))

## Debugging / fix

This may be a match for an upstream Flutter issue:

- https://github.com/flutter/flutter/issues/130976

Recently a PR was opened for that:

- https://github.com/flutter/flutter/pull/176492

so I'm hopeful the issue will be fixed once that's merged and we update Flutter.

**Comment by chrisbobbe** [comment]
Ah, the issue stopped reproducing when the user tested with an emoji that's known to have `still_url`. Many animated emoji in that org are missing `still_url`, presumably because they were uploaded before servers started recording that data.

Link to discussion (for those with access): [#zulip > Animated Reactji @ 💬](https://recurse.zulipchat.com/#narrow/channel/19176-zulip/topic/Animated.20Reactji/near/546126110)

The backend should fill in `still_url` for animated emojis that don't have it; I've raised this in [#issues > Animated image emoji without still_url @ 💬](https://chat.zulip.org/#narrow/channel/9-issues/topic/Animated.20image.20emoji.20without.20still_url/near/2281318). Since I thought we had a mobile-client bug where we fail to read Android's disable-animations setting, and there's no longer evidence for that, I'll close this issue.

Related issues:

- #966
- #1924
